### PR TITLE
feat(3000): help & support panel

### PR DESCRIPTION
**Pull request by corywatilo**
Support side panel now suggests people do some things before contacting us:

image

If they still need help and have email support, they can see what support response times are expected to be and submit an email:

image

If they are free users (no email support) then they see this instead of the option to email someone:

image

### CODE REVIEW

File: frontend/__snapshots__/lemon-ui-colors--color-palette--dark.png
Comment (raquelmsmith) [inline frontend/__snapshots__/lemon-ui-colors--color-palette--dark.png:1]: @corywatilo did you intend to make this color change?

File: unit.json [hunk 1]
```diff
@@ -19,11 +19,7 @@
         "posthog": [
             {
                 "match": {
-                    "uri": [
-                        "/_health",
-                        "/_readyz",
-                        "/_livez"
-                    ]
+                    "uri": ["/_health", "/_readyz", "/_livez"]
```
Comment (raquelmsmith) [inline unit.json:22]:
```suggestion
                    "uri": [
                        "/_health",
                        "/_readyz",
                        "/_livez"
                    ]
```

File: docker-compose.dev-full.yml [hunk 2]
```diff
@@ -182,4 +182,4 @@ services:
             - clickhouse
             - kafka
             - object_storage
-            - temporal
\ No newline at end of file
+            - temporal
```
Comment (raquelmsmith) [inline docker-compose.dev-full.yml:185]:
```suggestion
            - temporal
            
```

File: docker-compose.dev-full.yml [hunk 3]
```diff
@@ -182,4 +182,5 @@ services:
...
+
```
Comment (raquelmsmith) [inline docker-compose.dev-full.yml:186]:
```suggestion
            - temporal
```

File: frontend/src/layout/navigation-3000/components/AlgoliaSearch.tsx [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+    const { hits } = useHits()
+    const { items, refine } = useRefinementList({ attribute: 'type' })
+
+    const ref = useRef<HTMLDivElement>(null)
+    const [searchValue, setSearchValue] = useState<string>('')
+    const [activeOption, setActiveOption] = useState<undefined | number>()
+    const [activeTag, setActiveTag] = useState('all')
+    const [searchOpen, setSearchOpen] = useState(false)
+
+    const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>): void => {
```
Comment (Twixes) [inline frontend/src/layout/navigation-3000/components/AlgoliaSearch.tsx:164]: Nit: Would be really intuitive if ArrowLeft/Right switched between All/Docs/Questions/Tutorials

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 5]
```diff
@@ -1,51 +1,305 @@
...
 
-import { SidePanelTab } from '~/types'
+import { AvailableFeature, ProductKey, SidePanelTab } from '~/types'
 
+import AlgoliaSearch from '../../components/AlgoliaSearch'
 import { SidePanelPaneHeader } from '../components/SidePanelPaneHeader'
+import { SIDE_PANEL_TABS } from '../SidePanel'
 import { sidePanelStateLogic } from '../sidePanelStateLogic'
 
+const products = [
```
Comment (Twixes) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:31]: Global constants should be `CAPITALIZED`

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 6]
```diff
@@ -1,51 +1,305 @@
...
+                center
+                className="mt-2"
+            >
+                Cancel
+            </LemonButton>
+        </Section>
+    )
+}
+
 export const SidePanelSupport = (): JSX.Element => {
```
Comment (Twixes) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:140]: We need two Storybook stories here: 1. Free help, no form 2. Paid help, with form

**Review by Twixes — CHANGES_REQUESTED** [review]
Love the new panel. Just found a few snags

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx (hunk 6 above)
Comment (Twixes) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:140]: We already haves ones for all the other panels, so should be super simple to get done

File: frontend/src/layout/navigation-3000/components/AlgoliaSearch.tsx (hunk 4 above)
Comment (raquelmsmith) [reply frontend/src/layout/navigation-3000/components/AlgoliaSearch.tsx:164]: good idea, that sparks joy

**Pull request merged**